src/client/sandbox/xhr.ts

```diff
@@ -85,35 +84,33 @@ export default class XhrSandbox extends SandboxBase {
             });
         }
 
-        window.XMLHttpRequest           = xmlHttpRequestWrapper;
-        xmlHttpRequestWrapper.prototype = xmlHttpRequestProto;
-        xmlHttpRequestWrapper.toString  = () => xmlHttpRequestToString;
-
         // NOTE: We cannot just assign constructor property in OS X 10.11 safari 9.0
+        overrideConstructor(window, 'XMLHttpRequest', xmlHttpRequestWrapper);
```

Comment:
`// NOTE: We cannot just assign constructor property in OS X 10.11 safari 9.0`

Is this comment still actual?

Reply:
Yes, we still cannot just assign constructor property (as `overrideConstructor` does when the fourth parameter `overrideProtoConstructor` is set to true). In the code below this line, we are using 'objectDefineProperty' in order to override constructor property.

Reply:
I think a better way to phrase it to make the comment clearer would be:
```
// NOTE: We cannot just assign constructor property of the prototype of XMLHttpRequest in OS X 10.11 safari 9.0
```

Reply:
At present, we don't use the Safari 9.0 (Safari 11.3 and 12.0 versions are used).
If this code is necessary, then update comment to 
`// NOTE: We cannot just assign constructor property of the prototype of XMLHttpRequest starts from safari 9.0`